src/assets/config.default.json

```
@@ -5,7 +5,7 @@
     "name": "app",
     "remote_url": "http://localhost:4200/db/",
     "timeout": 60000,
-    "useTemporaryDatabase": true
+    "useTemporaryDatabase": false
```

```suggestion
    "useTemporaryDatabase": true
```

Let's keep the default config in the repository set to use the temporary database for testing? That makes it easier for new developers to get things running without the need for remote connections.

You can copy config.default.json to a new file "config.json" to overwrite your local config. The config.json file is on the gitignore list and won't be checked into git so everybody can keep their own custom settings locally